nengo/builder/tests/test_transforms.py

```diff
@@ -189,67 +189,68 @@ def test_convtransposeinc_2d(
     conv_transpose_output = AtCt.dot(x_flat)
     assert conv_transpose_output.shape == conv_output.shape
 
-    # Check whether the outputs match. If they don't, print/plot some diagnostic output.
     success = allclose(conv_transpose_output, conv_output)
-    if not success:
-        # transpose convolution should act as the derivative for forward convolution,
-        # compute and compare to the derivative to see exactly where the difference is
-        signals = {sig: np.array(sig.initial_value) for sig in (x, w, y)}
-        step_conv = ConvInc(w, x, y, conv).make_step(signals, None, None)
-        step_conv()
+    # if not success:
```

Comment:
Why comment this out?

Reply:
Whoops that was debugging to make sure I was passing in the right things to the debug function. That might explain why it's running out of memory actually ...